drivers/webdav/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="onitu-webdav",
+    version="0.1",
+    url="http://onitu.github.io",
+    description="Store files on a remote machine using webdav through onitu.",
+    license="MIT",
+    packages=find_packages(),
+    install_requires=["pycurl", "lxml", "webdavclient"],
```

Comment:
because they are needed by webdavclient but not installed by webdavclient. I don’t have the time to work on a pull request.=